PSReadLine/ReadLine.cs

```diff
@@ -275,6 +290,18 @@ private void PrependQueuedKeys(ConsoleKeyInfo key)
         /// </summary>
         /// <returns>The complete command line.</returns>
         public static string ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
+        {
+            // Use a default cancellation token instead of CancellationToken.None because the
+            // WaitHandle is shared and could be triggered accidently.
+            return ReadLine(runspace, engineIntrinsics, _defaultCancellationToken);
+        }
+
+        /// <summary>
+        /// Entry point - called from the PowerShell function PSConsoleHostReadLine
```

Comment:
I updated the comment with reflect intended usage.  I kept the comment generic because it can be used by more than just the PSES host, but that's right we aren't using `ConsoleHost`.  The way `ReadLine` is being called is more or less the same as in `ConsoleHost`, just a custom script instead of the `PSConsoleHostReadLine` function.